diff --git a/Cargo.toml b/Cargo.toml
index 6c7361f723..fec26de636 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -3825,17 +3825,6 @@ rustdoc-args = ["-Zunstable-options", "--generate-link-to-definition"]
 all-features = true
 cargo-args = ["-Zunstable-options", "-Zrustdoc-scrape-examples"]
 
-[[example]]
-name = "monitor_info"
-path = "examples/window/monitor_info.rs"
-doc-scrape-examples = true
-
-[package.metadata.example.monitor_info]
-name = "Monitor info"
-description = "Displays information about available monitors (displays)."
-category = "Window"
-wasm = false
-
 # Testbed
 [[example]]
 name = "testbed_2d"
